bridetalk_project/src/components/atoms/typographies/BodyText.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./bodytext.css";
+
+export const BodyText = (props) => {
```

Comment:
The whole Typography folder that you have, could be 1 component. 2 files (1 css, 1 js) instead of 10.
Imagine adding more and more typographies in ur project. Will u keep creating components for them?